modules/monitoring-exploring-the-visualized-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="exploring-the-visualized-metrics_{context}"]
+= Exploring the visualized metrics
+
+After running the queries, the metrics are displayed on the interactive plot. The X axis of the plot represents time. The Y axis represents the metrics values. Each metric is shown as a colored graph. You can manipulate the plot and explore the metrics.
+
+.Procedure
+
+. Initially, all metrics from all enabled queries are shown on the plot. You can select which metrics are shown.
+* To hide all metrics from a query, click the action button of the query and click "Hide all series".
```

Comment:
Based on our [guidelines](https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), I'd bold button names, such as _action_ button.

Reply:
You are right, implemented everywhere.